packages/typespec-vscode/test/manual/import-typespec-from-openapi3-test-plan.md

```diff
@@ -0,0 +1,67 @@
+# Import TypeSpec from OpenAPI3 Test Scenario _(stretch goal for SE)_
+
+With the TypeSpec emitter for OpenAPI3, users can import a TypeSpec file from a designated OpenAPI3 document. While it is possible to repeatedly convert OpenAPI3 to TypeSpec.
+
+## Test Environment
+
+* OS : Windows or Linux
+
+> Note: The extension should support all test cases in VS Code for Windows and Linux. Mac support is a stretch goal for Selenium semester.
+
+## Prerequisites
+
+Install TypeSpec Compiler before starting to write TypeSpec.
+
+* [Node.js 20+](https://nodejs.org/download/)
+* Npm 7+
+* [Install TypeSpec Compiler CLI](https://typespec.io/docs/): `"npm install -g @typespec/compiler"`
+
+## Test Steps
+
+### Step 1: Install the typespec extension.
```

Comment:
the user scenario in the spec is out-of-date. You can check the latest UI/User experience at https://github.com/microsoft/typespec/pull/5451